requirements.md

```diff
@@ -70,11 +48,46 @@ For external access to the Anbox Cloud Appliance, a couple of network ports must
 
 How to allow incoming traffic on the listed ports is different depending on the cloud used. Please consult the documentation of the cloud for further information on how to change the firewall.
 
+<a name="appliance-ubuntu-version"></a>
+### Ubuntu version
+
+The Anbox Cloud Appliance requires the following Ubuntu version:
+
+* 20.04 (focal)
+
 <a name="juju-based"></a>
 ## Juju-based deployments
 
 Anbox Cloud deployments are managed by Juju. They can be created on all the [supported clouds](https://juju.is/docs/clouds) as well as manually provided machines as long as they follow the required minimums.
 
+<a name="ac-ubuntu-version"></a>
+### Ubuntu version
+
+Anbox Cloud supports the following Ubuntu versions:
+
+* 18.04 (bionic) - see [Linux kernel](#linux-kernel) below for limitations
+* 20.04 (focal)
+* 22.04 (jammy)
+
+For new deployments, Ubuntu 22.04 (jammy) is preferred.
```

Comment:
For now, our CI only covers the following Ubuntu distros for the spread tests against deployment-scripts
- bionic
- focal

I don't think we run spread tests on the jammy for the deployment-scripts yet, @morphis Could you confirm this?